Library/Homebrew/diagnostic.rb

```diff
@@ -560,15 +560,15 @@ def check_for_symlinked_cellar
 
       def check_git_version
         # https://help.github.com/articles/https-cloning-errors
```

Comment:
Is this comment still relevant?

Reply:
@reitermarkus Yeh, probably not, thanks.